cinnabar/util.py

```diff
@@ -608,6 +609,20 @@ def run(func):
         reporter = MemoryReporter()
     try:
         retcode = func(sys.argv[1:])
+    except Exception as e:
+        # Catch all exceptions and provide a nice message
+        retcode = 1
+        sys.stderr.write(
+            'Sorry. The cinnabar extension exited because of an error.\n')
+        if check_enabled('traceback'):
+            traceback.print_exc()
+        else:
+            if hasattr(e, 'message') and e.message:
+                sys.stderr.write('{}\n'.format(e.message))
+
+            sys.stderr.write(
+                'Please add the `cinnabar.check = traceback` setting to the '
```

Comment:
Mmmm. I'm not sure advertising changing the git config is great. Moreover, if you edit your git configuration manually, you don't really write it this way. It's probably better to indicate how to do it for one command, which is to use `git -c cinnabar.check=traceback <command>`.